### DISCUSSION TITLE
docs/cli/run

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#134] [General]
# docs/cli/run

在使用GoFrame框架构建项目时，如何通过gf run命令实现自动编译功能。虽然Go语言本身不支持热编译特性，但gf run命令可实现当项目中的go文件发生变更时，自动编译并运行新版本程序的功能，旨在提高开发效率。

https://goframe.org.cn/docs/cli/run

**Comment by dabaode20000** [comment]
当修改了 manifest/config/config.yaml 的监听端口,但是启动服务时 gf run main..go 为什么服务监听的还是原来的端口
